mailpoet/tasks/phpstan/phpstan.neon

```diff
@@ -52,6 +52,14 @@ parameters:
       message: "#^Cannot cast string|void to string\\.$#"
       count: 5
       path: ../../lib/Automation/Engine/Storage/WorkflowStorage.php
+    -
+      message: '/^Call to static method custom_orders_table_usage_is_enabled\(\) on an unknown class Automattic\\WooCommerce\\Utilities\\OrderUtil\.$/'
+      count: 1
+      path: ../../lib/WooCommerce/Helper.php
+    -
+      message: '/^Call to function method_exists\(\) with/'
```

Comment:
I was not able to easily find how to match the whole error string and that is why I added only part of the string. I was running into regex parse errors when trying to add the whole string and escape special characters.

Reply:
It mentions the function `method_exists` and also it is defined for `../../lib/WooCommerce/Helper.php` so I would say it is specific enough.